vnext/ReactUWP/Modules/AppearanceModule.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <activeObject/activeObject.h>
+#include <cxxreact/CxxModule.h>
+#include <eventWaitHandle/eventWaitHandle.h>
+
+#include "IReactInstance.h"
+
+namespace react::uwp {
+
+// Listens for the current theme on the UI thread, storing the most recent. Will emit JS events on Appearance change.
+class AppearanceChangeListener final : public Mso::ActiveObject<> {
```

Comment:
This is a very weird `ActiveObject`. Initially I wanted to do more, but then I discovered:
1. The thread you construct `UISettings` on doesn't matter in terms of where it notifies you, and it itself is apparently freethreaded, so we don't delay construction until init
2. `m_currentTheme` was going to be an `ActiveReadableField` set by the UI thread, but adding a mutex felt like extra overhead when we could use an atomic for an integral type, so I didn't do that
3. `IReactInstance` is already free-threaded and didn't need protection it sounded like?

In the end, we basically just use `ActiveObject` to associate the UI Queue without really needing a lot of its features. I could this this making sense to just be a normal object keeping a queue instead, but don't think there would be too much of a difference.